data/test/scenarios/test_add_or_sub_in_leadership.cfg

```diff
@@ -0,0 +1,183 @@
+# This unit test is used for verify the work of abilities used like weapons, but also of special_id_active attribute.
+# Here, the teacher and student are the same unit
+{GENERIC_UNIT_TEST "test_add_in_leadership_abilities" (
+    [event]
+        name=start
+        [modify_unit]
+            [filter]
+            [/filter]
+            max_hitpoints=100
+            hitpoints=100
+            attacks_left=1
+        [/modify_unit]
+        [object]
+            silent=yes
+            [effect]
+                apply_to=new_ability
+                [abilities]
+                    [leadership]
+                        id=leadership_pos
+                        add=25
+                        cumulative=no
+                        affect_self=yes
+                    [/leadership]
+                    [attacks]
+                        value=1
+                    [/attacks]
+                    [damage]
+                        value=60
+                    [/damage]
+                    [chance_to_hit]
+                        id=test_cth
+                        value=100
+                    [/chance_to_hit]
+                [/abilities]
+            [/effect]
+            [filter]
+                id=alice
+            [/filter]
+        [/object]
+        # Place Alice for fighting
```

Comment:
This is a comment about the lines that follow it, but it has a blank-line between the comment and the code that's being commented on, combined with no blank line between it and the preceding code. That makes it harder to understand the code.

This comment applies to other parts of this code too.